tests/contrib/redis/test_redis_cluster.py

```diff
@@ -32,6 +32,13 @@ def tearDown(self):
         unpatch()
         super(TestRedisClusterPatch, self).tearDown()
 
+    @TracerTestCase.run_in_subprocess(env_overrides=dict(DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v1"))
+    def test_span_service_name_v1(self):
+        us = self.r.get("cheese")
+        spans = self.get_spans()
+        span = spans[0]
+        assert span.service == "unnamed-python-service"
```

Comment:
Trying to understand what this will look like down the road once implemented on all integrations. Essentially if you don't set `DD_SERVICE` or any other service naming, and did `DD_TRACE_SPAN_ATTRIBUTE_SCHEMA="v1"` then you'd end up with everything (web app, databases, messaging libraries) reporting data under `service:unnamed-python-service` correct?

Reply:
That is correct, but its only half the equation.  These changes will be released 'public announcement' in conjunction with updates for peer.service.

What this means: the current connection code here will report under 'service:unnamed-python-service' but each of the connections will have a child span with 'service:$peer.service'.